manifester/helpers.py

```diff
@@ -1,15 +1,24 @@
 """Defines helper functions used by Manifester."""
 from collections import UserDict
+import json
+import os
 from pathlib import Path
 import random
+import re
+import subprocess
+import sys
 import time
 
 from logzero import logger
 from requests import HTTPError
 import yaml
 
+from manifester.logger import setup_logzero
 from manifester.settings import settings
 
+setup_logzero(level="info")
```

Comment:
If switching to using the settings module, you can just use the configured log level